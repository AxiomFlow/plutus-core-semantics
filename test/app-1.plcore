(module Foo (imported) (exported () ()) (local () ())
  (define result
  (let (lam x (builtin addInt (x 5))) addFive [addFive 10]))
)
