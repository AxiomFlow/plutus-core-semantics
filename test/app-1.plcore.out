<T> <k> . </k> <env> ( Foo . result ) |-> 15 </env> <mod> Foo </mod> </T>
